config/locales/en.yml

```diff
@@ -37,6 +37,7 @@ en:
   keep_it_secret: Keep it secret, keep it safe!
   reset: Reset
   are_you_sure: Are you sure?
+  are_you_sure_cs: Warning! This will delete all Virtual Machine Templates which exist on this CS. Are you sure?
```

Comment:
May you say "which exist on this CS and are managed by Atmosphere"? But I guess that safer is this what you proposed.